src/pages/home/report/ReportActionCompose.js

```diff
@@ -135,6 +139,8 @@ class ReportActionCompose extends React.Component {
     }
 
     render() {
+        const {width} = Dimensions.get('window');
+        const isSmallScreenWidth = width <= variables.mobileResponsiveWidthBreakpoint;
```

Comment:
This will get the dimensions on every render but we only need them when the dimensions change.

Reply:
No problem

Reply:
This can be resolved by using `withWindowDimensions`